exe/main.cc

```diff
@@ -66,9 +66,12 @@ int startAgent(std::vector<std::string> arguments) {
     try {
         Agent agent { Configuration::Instance().getAgentConfiguration() };
         agent.start();
+    } catch (const Agent::PCPConfigurationError& e) {
```

Comment:
This new Error struct is used in the event that association fails - should it therefore be named PCPAssociationError or something similar? To a user or a support engineer; "PCP configuration" might suggest something in a config file. I think in this context "configuration" instead means the combination of PCP broker/client components?